### DISCUSSION TITLE
What happened to ubkp?

### DISCUSSION THREAD

**Discussion by JupiterRider** [raysan5/raylib#4001] [General]
ubkp was a raylib contributor, but I just noticed that a he turned into a ghost.

**Comment by raysan5** [comment]
@JupiterRider @asdqwe `ubkp` was a great raylib contributor, it made it possible raylib 5.0 and the platforms split... its contributions will prevail on raylib... It seems the user just decided to quit GitHub... it's sad but it's a legit personal decision... this network, like many other social networks, could be really time/life consuming...
